common/cards/card-plugins/single-use/mending.js

```diff
@@ -0,0 +1,63 @@
+import singleUseCard from './_single-use-card'
+import {GameModel} from '../../../../server/models/game-model'
+import {validPick} from '../../../../server/utils/reqs'
+
+/**
+ * @typedef {import('common/types/pick-process').PickRequirmentT} PickRequirmentT
+ *
+ */
+
+// TODO - Must work with Gemintay ability to use two single use cards per turn (should mend the first one)
+class MendingSingleUseCard extends singleUseCard {
+	constructor() {
+		super({
+			id: 'mending',
+			name: 'Mending',
+			rarity: 'ultra_rare',
+			description:
+				'Move any attached effect card from your active Hermit to an AFK Hermit.',
+		})
+
+		this.pickOn = 'apply'
+		this.pickReqs = /** @satisfies {Array<PickRequirmentT>} */ ([
+			{target: 'player', type: ['effect'], amount: 1, active: true},
+			{
+				target: 'player',
+				type: ['effect'],
+				amount: 1,
+				empty: true,
+				active: false,
+			},
+		])
+	}
+
```

Comment:
you also need a canAttach method that makes sure the active hermit has an effect card - see one of niko's merged signle use cards.